frontend/package.json

```diff
@@ -76,8 +76,10 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.12.0",
     "@types/react-color": "^3.0.1",
+    "@types/react-rte": "^0.16.1",
     "history": "^4.10.1",
     "immutability-helper": "^3.0.1",
-    "react-color": "^2.18.0"
+    "react-color": "^2.18.0",
+    "react-rte": "^0.16.1"
```

Comment:
Nice catch! I'll switch to this editor (initially I looked at Quill.js, but it didn't look as good).
I'll switch to this other editor, I as already found I few issues with `react-rte` aside from what you mentioned.

Btw, we're not supporting custom image upload, because it would be too hard to store and them pass this to the appserver. So I'll only implement the capability to embed externally hosted images.

Reply:
@Agrendalath I've found 2 good options:

1. https://github.com/ianstormtaylor/slate
2. https://github.com/zenoamaro/react-quill

Both are in beta, but so is the bootstrap version we're using.

I'll try both and see which fits better.

Reply:
Btw, the one you suggested also looks a bit abandoned. No commits since January.

Reply:
@Agrendalath Sigh, I'm having a bunch of trouble with every editor I try to fit in here :(

Reply:
Slate is too complex, and React Quill has a lot of quirks and weird bugs.

Reply:
Switched to TinyMCE, and it works wonders, has no state management bugs! 
Two downsides though: 
1. Unable to add snapshot tests (tinymce generates a random id and I didn't dig too deep on how to fix that).
2. We'll need to `eject` the project to include TinyMCE into the bundle, otherwise we'll get this pesky warning:
![Image Pasted at 2020-4-9 12-00 (1)](https://user-images.githubusercontent.com/27893385/78928328-cd75cb00-7a76-11ea-8e59-291ee7da4177.png)

Reply:
@Agrendalath I'll keep replying here for context.

>Does it mean that we would need to update everything for this package manually instead of using npm for this?

No we'll still use `npm` to manage packages as usual. The main change would be that we'll expose all webpack configuration and npm command scripts and store them in the repo.

This will allow us to tweak our project to better fit our needs, like enabling CSS Modules (not working because we need to add a loader to the WebPack configuration), and including additional dependencies in the bundle (like the TinyMCE imports).

For the TinyMCE change, we just need to add a few lines to `webpack.conf.js` (which is not accessible until we eject):
```javascript
      // TinyMCE
      // Must be accessible globaly for tinymce-react
      {
        test: require.resolve('tinymce/tinymce'),
        use: [ 'imports-loader?this=>window', 'exports-loader?window.tinymce' ]
      },
```

Reply:
> I'll keep replying here for context.

Ok, I'm seeing the "reply" section now. Must've been some weird GH bug.

Got it, sounds reasonable then :+1: